internal/exec/atlantis_generate_repo_config.go

```diff
@@ -443,14 +444,14 @@ func ExecuteAtlantisGenerateRepoConfig(
 	fileName := outputPath
 	if fileName == "" {
 		fileName = cliConfig.Integrations.Atlantis.Path
-		u.PrintInfo(fmt.Sprintf("Using 'atlantis.path: %s' from 'atmos.yaml'", fileName))
+		u.LogInfo(cliConfig, fmt.Sprintf("Using 'atlantis.path: %s' from 'atmos.yaml'", fileName))
 	} else {
-		u.PrintInfo(fmt.Sprintf("Using '--output-path %s' command-line argument", fileName))
+		u.LogInfo(cliConfig, fmt.Sprintf("Using '--output-path %s' command-line argument", fileName))
 	}
 
 	// If the path is empty, dump to 'stdout'
 	if fileName != "" {
-		u.PrintInfo(fmt.Sprintf("Writing atlantis repo config file to '%s'\n", fileName))
+		u.LogInfo(cliConfig, fmt.Sprintf("Writing atlantis repo config file to '%s'\n", fileName))
```

Comment:
> # Supported log levels: Off, Trace, Info
I think this, and many others, should be a `debug` level.  We should drastically reduce the majority of output from atmos. We make that hard by supporting only a single `info` or `error` level.